packages/components/src/DataList/components/DataListFilters/components/DataListSort/DataListSort.tsx

```diff
@@ -56,7 +44,14 @@ export function DataListSort() {
         const label = headers[sort];
         if (!label) return acc;
 
-        acc.push({ label, value: sort.toString() });
+        if (label === "Last activity") {
+          acc.push({ label: "Last Activity (Newest)", value: `${sort},desc` });
+          acc.push({ label: "Last Activity (Oldest)", value: `${sort},asc` });
```

Comment:
not particularly in love with this. Would be good to add a ticket to allow sorting to replace asc/desc labels per header

Reply:
Me neither. I'll add it